pkg/settlement/pseudosettle/mock/pseudosettle.go

```diff
@@ -62,7 +70,10 @@ func NewSettlement(opts ...Option) settlement.Interface {
 	return mock
 }
 
-func (s *Service) Pay(_ context.Context, peer swarm.Address, amount uint64) error {
+func (s *Service) Pay(c context.Context, peer swarm.Address, amount uint64) error {
+	if s.pay != nil {
+		return s.Pay(c, peer, amount)
```

Comment:
meant to be  `s.pay`  no?

Reply:
yes, tests worked anyway because I used the old mock for the tests which actually paid something. I switched all tests over to this one and removed the old mock.